ca/server.go

```diff
@@ -36,6 +36,10 @@ type Server struct {
 	joinTokens                  *api.JoinTokens
 	reconciliationRetryInterval time.Duration
 
+	// skipAutoUpdateRootCA, if set to true, causes the CA server to skip automatically updating
+	// the signing credentials based on changes in the memory store
+	skipAutoUpdateRootCA bool
```

Comment:
The auto-update functionality seems like it's only used for test cases. I'm curious how difficult/ugly it would be to completely remove this.